ios/Trifle/Sources/Trifle.swift

```diff
@@ -71,10 +71,11 @@ public class Trifle {
      - parameters: data - raw data to be signed.
      - parameters: keyHandle - key handle used for the signing.
      - parameters: certificate list - certificate chain to be included in the SignedData message.
-        Must match the key in keyHandle.
+     Must match the key in keyHandle.
 
      - returns:`SignedData` - signed data message in the Trifle format.
-    */
+     */
+    @available(*, deprecated)
```

Comment:
I think we can break the API since it would result in an compile time error when upgrading the library. i.e. we should just delete this

Reply:
That works for me. I was trying to do a soft remove with deprecating.